source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cpp

```diff
@@ -39,28 +65,6 @@ int main()
         };
         std::cout << settings.processing() << std::endl;
 
-        std::cout << "Configuring base acquisition with settings same for all HDR acquisition:" << std::endl;
-        const auto baseAcquisition = Zivid::Settings::Acquisition{ Zivid::Settings::Acquisition::Brightness{ 1.8 } };
-        std::cout << baseAcquisition << std::endl;
-
-        std::cout << "Configuring acquisition settings different for all HDR acquisitions" << std::endl;
-        const std::vector<double> aperture{ 8.0, 4.0, 4.0 };
-        const std::vector<double> gain{ 1.0, 1.0, 2.0 };
-        const std::vector<size_t> exposureTime{ 10000, 10000, 40000 };
-        for(size_t i = 0; i < aperture.size(); ++i)
-        {
-            std::cout << "Acquisition " << i + 1 << ":" << std::endl;
-            std::cout << "  Exposure Time: " << exposureTime.at(i) << std::endl;
-            std::cout << "  Aperture: " << aperture.at(i) << std::endl;
-            std::cout << "  Gain: " << gain.at(i) << std::endl;
-            const auto acquisitionSettings =
-                baseAcquisition.copyWith(Zivid::Settings::Acquisition::Aperture{ aperture.at(i) },
-                                         Zivid::Settings::Acquisition::Gain{ gain.at(i) },
-                                         Zivid::Settings::Acquisition::ExposureTime{
-                                             std::chrono::microseconds{ exposureTime.at(i) } });
-            settings.acquisitions().emplaceBack(acquisitionSettings);
-        }
```

Comment:
Note the comment here ("Configuring acquisition settings different for all HDR acquisitions"). The point was to show that base acquisition settings are modified here (using _copyWith_). We lost this demonstration in this PR.

Reply:
Perhaps we have to demonstrate this elsewhere, in settingsTutorial for example.

Reply:
@SatjaSivcev , CaptureHDRCompleteSEttings are now reverted to use the old method, but getting exposurevalues from a function based on which camera is connected. As you commented above, this way we still show different ways of using the API. 
Have we decided we will make a settingtutorial? The knowledge base is not sufficient? 
If we do it in a settingTutorial, similar to captureTutorial, we would still need to have a sample showing the code right? In the tutorials we just reference the code in samples (same as captureTutorial).

Reply:
We will make settingsTutorial (it is in the backlog). The Tutorials are purely demonstrating the SDK, KB explains all the settings but not all the ways one can configure them in the API.
Tutorials can but don't have to reference code samples. We can also hard code the code in the tutorials.